### PR TITLE
AB#5003

### PR DESCRIPTION
**Pull request by sachin755**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by johnjdl** [comment]
Working correctly in local build, @DallasP9124 @r-joyce please review and merge.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservation.js [hunk 1]
```diff
@@ -89,6 +90,9 @@ class RiskObservation extends Component {
             query={riskObservationLineQuery}
             variables={{ id: risk.id, first: 10, offset: 0 }}
             render={({ error, props }) => {
+              if (error) {
+                return toastGenericError('Request Failed');
```
Comment (BigBallard) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservation.js]: @sachin755 please continue to log errors in `console.error`. Refer to my comment in #287 please.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservationPopover.js [hunk 2]
```diff
@@ -124,6 +124,7 @@ class RiskObservationPopover extends Component {
       data,
       handleRemove,
     } = this.props;
+    console.log('RiskObservationPopover', data);
```
Comment (BigBallard) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/RiskObservationPopover.js]: Please do not include `console.log` statements in your PR's.

**Review by BigBallard — CHANGES_REQUESTED** [review]

**Comment by BigBallard** [comment]
@johnjdl don't forget to delete branches after merge 😉

**Pull request merged**